### ISSUE TITLE
Ensure that elements labelled through their content must have their visible text as part of their accessible name

### ISSUE THREAD

**Issue by annelfitz** [Esri/mountain-lion-habitat-analysis-js#14]
**Site:** https://esri.github.io/mountain-lion-habitat-analysis-js/
**Date scanned:** 09/15/2026

### Rule
**Rule ID:** label-content-name-mismatch
**Rule Name:** Elements must have their visible text as part of their accessible name
**Rule Description:** Ensure that elements labelled through their content must have their visible text as part of their accessible name
**Standard(s):** WCAG 2.1(A)
**WCAG Success Criteria:** 2.5.3 Label in Name (A) , EN-301-549 , EN-9.2.5.3
**Impact:** serious
https://dequeuniversity.com/rules/axe/4.11/label-content-name-mismatch?application=axeAPI

### Issue 1
- **Element location:** calcite-list-item[value="habitatAnalysis"] > .content-container--selectable.content-container--has-center-content[aria-label="Combined analysis"]
- **Element source:** <div role="gridcell" aria-label="Combined analysis" class=" grid-cell content-container content-container--selectable content-container--has-center-content ">
- **Suggested fix:** Fix any of the following:
  Text inside the element is not included in the accessible name

![Screenshot](https://github.com/Esri/mountain-lion-habitat-analysis-js/raw/allyhawk-assets/.allyhawk/screenshots/7f32b5485b50aac0c5fe9f134f7662a3.jpg)

**Comment by macandcheese** [comment]
Hm, the List Items are labelled.. I do notice the Chips are not labelled though for the measurement sliders.
